dictionaries/condensed-strokes.json

```diff
@@ -22,6 +22,7 @@
 "-B/HO*ELD": "behold",
 "-B/HO*LD": "behold",
 "-F/TP-PL": "of.",
+"-GT": "thing",
```

Comment:
@paulfioravanti what's the particular motivation for adding this to the condensed strokes dictionary? While it is indeed a condensed stroke that already works with existing Plover dictionary, it doesn't appear to follow any existing convention for phrasing briefs ending in GT and " thing". As it is shorter than the current preferred phonetic entry in the top X words dictionaries, Typey Type would prioritise the shorter entry, but I think the phonetic entry might be better to teach new steno students.

Reply:
> condensed stroke that already works with existing Plover dictionary
Yep, that was pretty much the only reason I included it: I inadvertently stumbled on it, and there was no named entry for it in Plover.

> I think the phonetic entry might be better to teach new steno students
I can whole-heartedly agree with this, and wouldn't want to have what could be considered an unintuitive entry be prioritised by Typey Type. Will remove it :+1: